app/script_variant.py

```diff
@@ -54,14 +63,15 @@ def romanize_japanese_location(word):
     if not word:
         return word
 
-    if word in JAPANESE_LOCATION_DICTIONARY:
-        yomigana = JAPANESE_LOCATION_DICTIONARY[word]
-        return jautils.hiragana_to_romaji(yomigana)
+    if is_kanji(word):
```

Comment:
By the way, why do you have is_kanji here but not in romanize_japanese_name_by_name_dict?